src/chocolatey.resources/helpers/chocolateyScriptRunner.ps1

```diff
@@ -52,17 +52,13 @@ $lastExecutableExitCode = $LASTEXITCODE
 
 if ($lastExecutableExitCode -ne $null -and $lastExecutableExitCode -ne '') {
   Write-Debug "The last executable that ran had an exit code of '$lastExecutableExitCode'."
-} 
+}
 
 if (-not $scriptSuccess) {
  Write-Debug "The script exited with a failure."
-} 
-
-$exitCode = 0
-if ($env:ChocolateyCheckLastExitCode -ne $null -and $env:ChocolateyCheckLastExitCode -eq 'true' -and $lastExecutableExitCode -ne $null -and $lastExecutableExitCode -ne '') {
-  $exitCode = $lastExecutableExitCode

```

Comment:
I am 85% sure the answer to this is that we don't have to worry, but just to be sure: Are there any changes in chocolatey.extension that will need to be made for this?

Reply:
That is a great question! I will take a look at that to confirm. This wouldn't stop this PR from going in, but potentially something we need to update over there!